src/pages/VoucherRedemption/Amount.tsx

```diff
@@ -38,7 +43,7 @@ const Amount = (props: Props) => {
 
   const handleAmount = (e) => {
     setError('');
-    if (e.target.value * 100 > voucher.amount)
+    if (e.target.value * 100 > voucher.amount || e.target.value * 100 < 5)
```

Comment:
Why 5 cents?

Reply:
Honestly up to you! I saw @bvillaroman had a minValue of 5 (cents) before, so I left it at that. This could be 1¢, $1, or whatever we want

Reply:
Changed this to 1 cent!